### PR TITLE
Slf4j

### PR DESCRIPTION
**Pull request by Trimax**
1. LoggerFactory.getLogger -> Slf4j annotation
2. Some warning fixes

### CODE REVIEW

File: pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/util/DataLoader.java [hunk 1]
```diff
@@ -365,16 +364,16 @@ void loadAboutYouSurveyData(Handle handle,
         AboutYouSurvey aboutYouSurvey = null;
         DatstatSurveyData surveyData = participantData.getParticipantUser().getDatstatsurveydata();
         if (surveyData != null) {
-            //LOG.info("survey data loaded..");
+            //log.info("survey data loaded..");
```
Comment (bskinner) [inline pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/util/DataLoader.java]: Let's remove this line entirely.

File: pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/util/DataLoaderMain.java [hunk 2]
```diff
@@ -193,7 +191,7 @@ public static void initDbConnection() {
 
         String dbUrl = cfg.getString(ConfigFile.DB_URL);
 
-        LOG.info("Initializing db pool for " + dbUrl);
+        log.info("Initializing db pool for " + dbUrl);
```
Comment (bskinner) [inline pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/util/DataLoaderMain.java]: Another spot where we're logging the database url. You know the drill- let's remove it! :)

File: pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/util/DataLoaderMain.java [hunk 3]
```diff
@@ -249,11 +247,11 @@ public void processGoogleBucketParticipantFiles(Config cfg, String studyGuid, bo
         for (Blob file : bucket.list().iterateAll()) {
             data = new String(file.getContent());
             if (!file.getName().startsWith("Participant_") && !file.getName().startsWith("LovedOne_")) {
-                LOG.info("Skipping bucket file: {}", file.getName());
+                log.info("Skipping bucket file: {}", file.getName());
                 continue;
                 //not a participant data .. continue to next file.
             }
-            LOG.info("File Name: {} \n DATA:\n", file.getName(), data);
+            log.info("File Name: {} \n DATA:\n", file.getName(), data);
```
Comment (bskinner) [inline pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/util/DataLoaderMain.java]: The `data` object is unused, let's remove it.
```suggestion
            log.info("File Name: {}", file.getName());
```

File: pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/util/DataLoaderMain.java [hunk 4]
```diff
@@ -522,13 +520,13 @@ public void processGoogleBucketMailingListFiles(Config cfg, String studyGuid) th
                 continue;
                 //not a mailing data .. continue to next file.
             }
-            LOG.info("File Name: {} \n DATA:\n", file.getName(), data);
+            log.info("File Name: {} \n DATA:\n", file.getName(), data);
```
Comment (bskinner) [inline pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/util/DataLoaderMain.java:523]: Unused `data` object- let's remove everything after the first placeholder.

File: pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/util/MigratedDataReconcileCli.java [hunk 5]
```diff
@@ -274,7 +272,7 @@ private void doCompare(CSVRecord csvRecord, Map<String, JsonElement> userData, M
                 }
             }
             consentSurveyName = consentSurveyName.concat("_v").concat(consentVersion);
-            //LOG.info("consent survey name: {} .. consent version: {}", consentSurveyName, consentVersion);
+            //log.info("consent survey name: {} .. consent version: {}", consentSurveyName, consentVersion);
```
Comment (bskinner) [inline pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/util/MigratedDataReconcileCli.java]: Let's remove this line entirely

File: pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/util/MigratedDataReconcileCli.java [hunk 6]
```diff
@@ -299,7 +297,7 @@ private void doCompare(CSVRecord csvRecord, JsonElement sourceDataEl, JsonElemen
         for (JsonElement thisMapData : dataArray) {
             sourceFieldName = thisMapData.getAsJsonObject().get("source_field_name").getAsString();
             targetFieldName = thisMapData.getAsJsonObject().get("target_field_name").getAsString();
-            //LOG.info("checking source field: {} ... target field: {} ", sourceFieldName, targetFieldName);
+            //log.info("checking source field: {} ... target field: {} ", sourceFieldName, targetFieldName);
```
Comment (bskinner) [inline pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/util/MigratedDataReconcileCli.java]: Let's remove the line.

File: pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/util/MigratedDataReconcileCli.java [hunk 7]
```diff
@@ -311,7 +309,7 @@ private void doCompare(CSVRecord csvRecord, JsonElement sourceDataEl, JsonElemen
             switch (sourceDataType) {
                 case "Date":
                     sourceFieldValue = getStringValueFromElement(sourceDataEl, sourceFieldName);
-                    //LOG.info("source field: {} value: {} targetField: {} target value: {}", sourceFieldName, sourceFieldValue,
+                    //log.info("source field: {} value: {} targetField: {} target value: {}", sourceFieldName, sourceFieldValue,
```
Comment (bskinner) [inline pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/util/MigratedDataReconcileCli.java]: Let's remove the line.

File: pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/util/MigratedDataReconcileCli.java [hunk 8]
```diff
@@ -351,7 +349,7 @@ private void doCompare(CSVRecord csvRecord, JsonElement sourceDataEl, JsonElemen
                     } else if (dkSet.contains(sourceFieldValue) && dkSet.contains(targetFieldValue)
                             && !sourceFieldName.contains("country")) {
                         //dk/DK/Don't know .. consider as match & move on
-                        //LOG.info("source field Name: {} .. target field Name: {} .. source field Value: {} .. target field Value: {}  ",
+                        //log.info("source field Name: {} .. target field Name: {} .. source field Value: {} .. target field Value: {}  ",
                         //        sourceFieldName, targetFieldName, sourceFieldValue, targetFieldValue);
```
Comment (bskinner) [inline pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/util/MigratedDataReconcileCli.java]: Let's remove this.

**Review by bskinner — DISMISSED** [review]
Looks good to me- just a couple of minor tweaks.

**Review by alextanlan — APPROVED** [review]

**Pull request merged**